app/models.py

```diff
@@ -97,6 +97,7 @@ class Meta:
 class WorkerLog(models.Model):
     update = models.DateField()
     source = models.CharField(max_length=10, choices=SOURCE_CHOICES)
+    tenders_count = models.IntegerField()
```

Comment:
def get_keywords_setting():
        return re.findall(r'[^,;\s]+', settings.TENDER_KEYWORDS)

Change to use Keyword model or if not possible drop the function from the model and implement it in the views.